backend/src/test/java/com/redhat/cloud/notifications/routers/TestEmailEndpointMigrationService.java

```diff
@@ -38,6 +39,7 @@
     Mutiny.Session session;
 
     @Test
+    @Disabled
```

Comment:
`EmailEndpointMigrationService` and `TestEmailEndpointMigrationService` must be removed before this PR is merged.